action/action.rb

```diff
@@ -105,10 +109,11 @@ def report_offense?(offense, change_ranges:)
 rescue Exception => e
   puts e.message
   puts e.backtrace.inspect
-  resp = check_run.error(message: e.message)
-  p resp
-  p resp.json
+  abort("::error:: #{e.message}")
 else
-  check_run.update(annotations: annotations)
   ActionUtils.set_output("issuesCount", annotations.count)
+  annotations.each do |note|
+    puts note.to_output_command
```

Comment:
Do you think it's worth using `ActionUtils` to format out annotation output?

Reply:
Currently ActionUtils just has a utility to echo debug commands and to set the output of the workflow entirely (like with annotationCount) not to "output" these types of commands. We could move the logic (or extract a portion) from `Annotation#to_output_command` but I think I would prefer to leave it the way it is — I'm only like 3/10 on that though.